Android/app/build.gradle

```diff
@@ -6,9 +6,9 @@ android {
         applicationId "com.example.yello.common"
         minSdkVersion 27
         targetSdkVersion 27
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
```

Comment:
# of chars.

Reply:
Sorry.... I confused that. I modified all of them.

Reply:
But I can't modify gradlew file in Window... So I modifed files except gradlew.